examples/caching/app.js

```diff
@@ -21,7 +21,7 @@ function loadNext() {
 
     count += chunkSize;
     if (count >= total) {
-      list.complete();
+      list.cache();
```

Comment:
This changed to be more explicit. Should make the user feel like they have more control over the cache.